### PR TITLE
opencv: Fixed correspondence not requiring version on Windows.

### PR DESCRIPTION
**Pull request by icedream2linxi**
Specify library name and version:  **opencv/3.4.17**

In Windows, there is no need for version correspondence, multiview and numeric libraries.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by SpaceIm — DISMISSED** [review]
4.x branch would also need this fix

**Comment by icedream2linxi** [comment]
> 4.x branch would also need this fix

Ok. I've fixed it.

File: recipes/opencv/4.x/conanfile.py [hunk 1]
```diff
@@ -713,8 +728,10 @@ def package_info(self):
         def get_lib_name(module):
             if module == "ippiw":
                 return "%s%s" % (module, debug)
-            prefix = "" if module in ("correspondence", "multiview", "numeric") else "opencv_"
-            return "%s%s%s%s" % (prefix, module, version, debug)
+            elif module in ("correspondence", "multiview", "numeric"):
+                return module
```
Comment (SpaceIm) [inline recipes/opencv/4.x/conanfile.py:732]: are you sure they don't have debug postfix?

Comment (icedream2linxi) [reply recipes/opencv/4.x/conanfile.py:732]: The picture is the result of my compilation with this parameter.
`conan create -k . opencv/4.5.5@ -o opencv:contrib=True -o opencv:contrib_sfm=True -o opencv:with_ipp=opencv-icv -s build_type=Debug --build=missing`
image

**Review by SpaceIm — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**